sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -1240,9 +1243,9 @@ static int solve_l2r_l1l2_svr(
 //
 // See Algorithm 5 of Yu et al., MLJ 2010
 
-#undef GETI
-#define GETI(i) (y[i]+1)
-// To support weights for instances, use GETI(i) (i)
+
+#define GETC(i) upper_bound[y[i]+1]*sample_weight[i]
+// To support weights for instances (sample_weight*class_weight), use GETC(i) (i)
```

Comment:
Oh right, the comment explains the macro. It's a bit strangely worded (not your fault, the original comment was weird).